lib/public/components/common/popover/overflowBalloon.js

```diff
@@ -23,7 +23,8 @@ import { PopoverAnchors } from './PopoverEngine.js';
  * @param {HTMLElement} element the element to look for overflow
  * @returns {boolean} true if the element overflow
  */
-const doesElementOverflow = (element) => Boolean(element) && element.offsetWidth < element.scrollWidth;
+const doesElementOverflow = (element) =>
+    Boolean(element) && element.offsetWidth < element.scrollWidth || Boolean(element) && element.offsetHeight < element.scrollHeight;
```

Comment:
Please don't put newline after `=>`. In this case, I think the code would be more clear with a body and intermediate variables `overvlowVertically` and `overflowHorizontally`